<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>searchform</class>
    <widget class="QWidget" name="searchform">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>430</width>
                <height>116</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>Search</string>
        </property>
        <layout class="QGridLayout" name="gridLayout">
            <item row="1" column="1">
                <widget class="QPushButton" name="pushButton">
                    <property name="text">
                        <string>Search</string>
                    </property>
                </widget>
            </item>
            <item row="0" column="0" colspan="2">
                <widget class="QLineEdit" name="lineEdit"/>
            </item>
            <item row="1" column="0">
                <widget class="QCheckBox" name="checkHex">
                    <property name="text">
                        <string>hex</string>
                    </property>
                </widget>
            </item>
        </layout>
    </widget>
    <tabstops>
        <tabstop>lineEdit</tabstop>
        <tabstop>checkHex</tabstop>
        <tabstop>pushButton</tabstop>
    </tabstops>
    <resources/>
    <connections/>
</ui>
